sql-statements/sql-statement-rename-index.md

```diff
@@ -83,7 +83,7 @@ Create Table: CREATE TABLE `t1` (
 ## 另请参阅
 
 * [SHOW CREATE TABLE](/sql-statements/sql-statement-show-create-table.md)
-* [CREATE INDEX](/sql-statements/sql-statement-create-index.md)
+* [CREATE INDEXS](/sql-statements/sql-statement-create-indexes.md)
 * [DROP INDEX](/sql-statements/sql-statement-drop-index.md)
 * [SHOW INDEX](/sql-statements/sql-statement-show-index.md)
```

Comment:
```suggestion
* [SHOW INDEXES](/sql-statements/sql-statement-show-indexes.md)
```

Reply:
sync change file sql-statement-show-index.md name to sql-statement-show-indexes.md ?